packages/patternfly-4/react-core/src/components/Chip/ChipWrapper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Can we use typescript for this component?

Reply:
I thought `ChipWrapper.d.ts` was good enough. Can you give me an example of doing that? Like rename the file from `ChipWrapper.js` to `ChipWrapper.ts` or something more complicated?

Thanks :smile_cat:

Reply:
The typescript stuff isn't fully baked yet for PF4 so I think your are right to just include the d.ts file. Eventually we will want more but for now this should be fine.

Reply:
Cool :sunglasses: . At any rate, if I can help with that at any point let me know.